src/raw.js

```diff
@@ -26,7 +27,8 @@ assign(Raw.prototype, {
   set: function(sql, bindings) {    
     this._cached  = undefined
     this.sql      = sql
-    this.bindings = bindings
+    this.bindings = _.isObject(bindings) || _.isUndefined(bindings) ?  bindings : [bindings]
```

Comment:
I would like to add parenthesis here for clarity... 

I was reading the line first the way that in case when _.isObject is true, true would be assigned to `this.bindings` since part after `||` won't need to be ran :) looks like the ternary operator makes the difference here...